File: app/src/main/java/com/rafamatias/nlp/domain/mapper/SentimentMapper.java
package com.rafamatias.nlp.domain.mapper;

import com.rafamatias.nlp.data.entity.SentimentResponse;
import com.rafamatias.nlp.presentation.model.SentimentModel;

public class SentimentMapper {

    public static SentimentModel fromSentiment(SentimentResponse sentiment) {
        return new SentimentModel(sentiment.getScore());
    }

}

File: domain/src/main/java/com/shrpereira/tweetsense/domain/mapper/SentimentMapper.kt
package com.shrpereira.tweetsense.domain.mapper

import com.shrpereira.tweetsense.data.remote.sentiment.SentimentResponse
import com.shrpereira.tweetsense.domain.model.SentimentModel

class SentimentMapper {

	fun map(response: SentimentResponse?): SentimentModel? {
		response?.let {
			return SentimentModel(it.documentSentiment.score ?: 0.0)
		}

		return null
	}
}